components/director/internal/domain/application/service.go

```diff
@@ -479,6 +498,17 @@ func (s *service) DeleteLabel(ctx context.Context, applicationID string, key str
 	return nil
 }
 
+func (s *service) deleteDefaultEventing(ctx context.Context, tenantID, applicationID string, scenariosToRemove []string) error {
+	appScenarios, err := s.scenariosService.GetScenarioNamesForApplication(ctx, applicationID)
+	if err != nil {
+		return err
+	}
```

Comment:
This is not needed, since you have it in the function that calls this one, you can just pass it as an argument.

Reply:
More comments below you should pass the input to this function and remove this call.